backend/mentorship/tests/test_cannot_delete_accepted_req.py

```diff
@@ -0,0 +1,101 @@
+import json
```

Comment:
Made some general changes, and tried to fix any linting issues, please review it once and still let me know if I'm missing any python guidelines. Also please let me know exactly what python guidelines did you refer to , so that I don't make the same mistakes again.